app/controllers/hackdays_controller.rb

```diff
@@ -0,0 +1,48 @@
+class HackdaysController < ApplicationController
+	
+	def create
+		@hackday = Hackday.new(hackday_params)
+		if @hackday.save
+			flash[:success] = "successfully added a hackday"
+			redirect_to @hackday
+		else
+			flash[:danger] = "failed to add a hackday"
```

Comment:
should use @hackday.errors.full_messages to give real validation errors.

also generally this would be flash[:errors] not :danger